src/_includes/landing/communities.njk

```diff
@@ -29,7 +29,7 @@
     </div>
   </div>
 </section>
-<section class="bg-white px-4 md:px-8 lg:px-16">
+<section class="bg-[#F0EEFF] px-4 md:px-8 lg:px-16">
```

Comment:
If you're using multiple times, please define it as a utility class.

Reply:
Done, added utility class bg-lavender